libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -873,10 +873,36 @@ void AP_GPS::update(void)
 #endif
 }
 
+/*
+    checks to see if the GPSs have all aquired a fix.
+*/ 
+bool AP_GPS::check_gps_warmup(void) {
+    // Previously warmed up.
+    if (_degraded_gps_enabled) {
+        return true;
+    }
+
+    // Once all GPSs have warmed up, enable the degraded GPS feature
+    for (uint8_t i = 0; i < GPS_MAX_RECEIVERS; ++i) {
+        if (state[i].status < GPS_OK_FIX_3D) {
+            return false; // GPS i hasn't warmed up yet, don't enable gps degredation.
+        }
+    }
+
+    // All GPSs have warmed up, feature can be enabled.
+    _degraded_gps_enabled = true;
+    return true;
+}
+
 /*
     determines if any gps's are degraded
 */
 uint8_t AP_GPS::update_and_count_degraded_gps(void) {
+    // Prior to the GPS being warmed up, nothign is considered degraded.
```

Comment:
nothign --> nothing

Reply:
Also, this is from the last PR and I didn't catch it then but can you please change all instances of "degreded" to "degraded"? Thank you!